.circleci/config.yml

```diff
@@ -0,0 +1,95 @@
+# Python CircleCI 2.0 configuration file
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+      # CircleCI maintains a library of pre-built images documented at
+      # https://circleci.com/docs/2.0/circleci-images/
+      # use `-browsers` prefix for selenium tests, e.g. `<image_name>-browsers`
+
+      # Python
+      - image: circleci/python:2.7.13
+        environment:
+          TZ: America/New_York
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - fec-eregs-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements_dev.txt" }}-{{ checksum "requirements_test.txt" }}-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - fec-eregs-dependencies-
+      
+      - run:
+          name: Install Ruby and SASS
+          command: |
+            sudo apt-get update && sudo apt-get install ruby-full
+            sudo gem install sass -v 3.4.25
+            export PATH="/usr/bin/ruby:$PATH"
+            export PATH="/usr/local/bin/sass:$PATH"
+
+      - run:
+          name: Install node dependencies
+          command: |
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.6/install.sh | bash
+            echo ". ~/.nvm/nvm.sh" >> $BASH_ENV
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+            [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+            nvm install 8.9.1
+            nvm use 8.9.1
+            nvm alias default 8.9.1
+            npm install
+            npm install -g grunt-cli bower
+            npm run build
+
+      - run:
+          name: Install Python dependencies
+          command: |
+            pip install virtualenv
+            virtualenv venv
+            source venv/bin/activate
+            pip install -r requirements.txt
+            pip install -r requirements_test.txt
+            pip install -r requirements_dev.txt
+
+      - save_cache:
+          paths:
+            - ./venv
+            - ./node_modules
+          key: fec-eregs-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements_dev.txt" }}-{{ checksum "requirements_test.txt" }}-{{ checksum "package.json" }}
+
+      - run:
+          name: Run tests
+          command: |
+            source venv/bin/activate
+            npm run build
+            python manage.py compile_frontend
```

Comment:
Which of these actually run the tests?  I'd expect to see `npm run test` or `npm run test-single`, and/or `python manage.py test`; this of course depends on the project setup, so I just want to double check since I'm not as familiar with this project.

Reply:
That is the other issue- the tests are broken- There were so many moving pieces that I wanted to get the deploy aspect working and then work on the other problems. With the upgrade next then, see what pugs remained and tackle those last.I wanted to make sure this was reviewed to make sure I was not playing too fast and loose with the merge.

Reply:
Ah okay, totally understand!  Definitely not something to address as a part of this then, let's get this in first!